src/main/java/org/ballerinalang/command/util/ToolUtil.java

```diff
@@ -800,4 +804,15 @@ public static String getTypeName(String version) {
             return preview;
         }
     }
+
+    private static void setPermission(String filePath) {
```

Comment:
shall we use `addExecutablePermissionToDirectory` to make consistent with other method?